sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -347,6 +349,7 @@ public partial class BlobContainerProperties
         public Azure.ETag ETag { get { throw null; } }
         public bool? HasImmutabilityPolicy { get { throw null; } }
         public bool? HasLegalHold { get { throw null; } }
+        public bool IsVersionLevelWormEnabled { get { throw null; } }
```

Comment:
maybe HasVersionLevelWorm ?

Reply:
We probably need to rename this property in general, since Version Level Worm is an internal name.

Reply:
Renamed to `HasVersionLevelWorm`